docs/library/usage/contributing.md

```diff
@@ -18,7 +18,7 @@ If you're thinking of adding a new feature, before starting any work, we recomme
 questions and get feedback from the maintainers and the community before you sink a lot of time into writing (possibly
 the wrong) code. If there is anything you’re unsure about, just ask!
 
-If you're submitting a simple change such as a new variable or new output value, it may be worth opening a pull request directly, as described below. The key point is that we want to minimize the risk that you invest time in something we ultimately can't use.
+If you're submitting a simple change such as a new variable or new output value, it may be worth opening a pull request directly, as described below.
```

Comment:
@josh-padnick Did this autogenerate again for some reason, or did you modify this file manually?

Reply:
Sorry about the confusion. I tried to address this in my PR initial comment but I don't think it was clear. When I submitted https://github.com/gruntwork-io/docs/pull/889, I assumed that if I submitted changes to `_docs-sources` folders, the build would auto-generate these files; I didn't realize we were auto-generating these locally and then committing them.

So in this PR, I attempted to commit the generated files.

Reply:
ahhh okay no worries!